packages/react-components/react-tabster/src/index.ts

```diff
@@ -32,12 +32,18 @@ export type {
 } from './focus/index';
 
 export { applyFocusVisiblePolyfill } from './focus/index';
-import { Types as TabsterTypes, dispatchGroupperMoveFocusEvent, dispatchMoverMoveFocusEvent } from 'tabster';
+import {
+  Types as TabsterTypes,
+  Events as TabsterEvents,
+  dispatchGroupperMoveFocusEvent,
+  dispatchMoverMoveFocusEvent,
+} from 'tabster';
 
 export type TabsterDOMAttribute = TabsterTypes.TabsterDOMAttribute;
 
 export type { KeyborgFocusInEvent } from 'keyborg';
 export { KEYBORG_FOCUSIN } from 'keyborg';
 
 // @internal (undocumented)
-export { TabsterTypes, dispatchGroupperMoveFocusEvent, dispatchMoverMoveFocusEvent };
+// eslint-disable-next-line deprecation/deprecation
+export { TabsterTypes, TabsterEvents, dispatchGroupperMoveFocusEvent, dispatchMoverMoveFocusEvent };
```

Comment:
This is wrong as only for deprecated APIs lint should be turned on, otherwise we have no idea what's what. please change this.

```suggestion
export { 
TabsterTypes, 
TabsterEvents, 
dispatchGroupperMoveFocusEvent, 
// eslint-disable-next-line deprecation/deprecation
dispatchMoverMoveFocusEvent 
};
```

Reply:
updated!